libs/api/domains/social-insurance/src/lib/socialInsurance.service.ts

```diff
@@ -0,0 +1,91 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import type { Logger } from '@island.is/logging'
+import { SocialInsuranceAdministrationClientService } from '@island.is/clients/social-insurance-administration'
+import { User } from '@island.is/auth-nest-tools'
+import { PaymentPlan } from './models/paymentPlan.model'
+import { handle404 } from '@island.is/clients/middlewares'
+import { PaymentGroup } from './models/paymentGroup'
+import { isDefined } from '@island.is/shared/utils'
+
+@Injectable()
+export class SocialInsuranceService {
+  constructor(
+    @Inject(LOGGER_PROVIDER) private readonly logger: Logger,
+    private readonly socialInsuranceApi: SocialInsuranceAdministrationClientService,
+  ) {}
+
+  async getPaymentPlan(
+    user: User,
+    year?: number,
+  ): Promise<PaymentPlan | undefined> {
+    const res = await this.socialInsuranceApi
+      .getPaymentPlan(user, year ?? 2023)
```

Comment:
default to `2023` if nothing is passed? Is there any logic behind that? Would it be better to get current year - 1?